docs/dasharo-menu-docs/overview.md

```diff
@@ -129,7 +129,11 @@ possible feature extension of your platform.
 
 User Password Management menu allows one to set firmware setup admin password:
 
+<<<<<<< HEAD
```

Comment:
@wiktormowinski yeah this shouldn't be here please fix

Reply:
idk how did it even get past through both `rebase --continue` and `pre-commit` X.X